index.html

```diff
@@ -15,31 +15,294 @@
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+            </div>
+            <div class="cards">
+                <img src="/Images/Img Plaholder.png" alt="Store image">
+                <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                    <div class="Languages">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="aboutme-button">See Project</button>
+                    </div>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+
+        </section>
+        <section class="About-me">
+            <div class="aboutme-text">
+                <div class="about-title">
+                    <h1>About me</h1>
```

Comment:
- [ ] There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make only your main title of " Hey there. I'm, Tite Lunda I'm a software developer" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] please kindly make changes and replace all other `h1` except your main title